tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -751,6 +807,100 @@ describe('Test application deployment based on clusterGroup', { tags: '@p1'}, ()
           cy.removeClusterLabels(dsCluster, key, value);
         }
       )
+      // Delete clusterGroups.
+      cy.deleteClusterGroups();
+    })
+  )
+
+  qase(29,
+    it("Fleet-29: Test install app to new set of clusters from old set of clusters using 'clusterGroup'", { tags: '@fleet-29' }, () => {
+      const repoName = 'default-single-app-cluster-group-29'
+      const new_key = 'key_third_cluster'
+      const new_value = 'value_third_cluster'
+      const newClusterGroupName = 'cluster-group-env-third-cluster'
+
+      cy.accesMenuSelection('Continuous Delivery', 'Clusters');
+      cy.contains('.title', 'Clusters').should('be.visible');
+
+      // Check application status on both clusters i.e. imported-0 and imported-1
+      dsFirstTwoClusterList.forEach(
+        (dsCluster) => {
+          cy.assignClusterLabel(dsCluster, key, value);
+        }
+      )
+
+      // Create group of cluster consists of same label.
+      cy.clickNavMenu(['Cluster Groups']);
+      cy.contains('.title', 'Cluster Groups').should('be.visible');
+      cy.createClusterGroup(clusterGroupName, key, value, bannerMessageToAssert);
+
+      // Create a GitRepo targeting cluster group created.
+      cy.addFleetGitRepo({ repoName, repoUrl, branch, path, deployToTarget: clusterGroupName });
+      cy.clickButton('Create');
+      cy.checkGitRepoStatus(repoName, '1 / 1');
+
+      // Check first application status on both clusters i.e. imported-0 and imported-1
+      dsFirstTwoClusterList.forEach((dsCluster) => {
+        cy.checkApplicationStatus(appName, dsCluster, 'All Namespaces');
+      })
+
+      // Add label to the third cluster
+      cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
+      cy.clickNavMenu(['Clusters']);
+      cy.contains('.title', 'Clusters').should('be.visible');
+      cy.assignClusterLabel(dsThirdClusterName, new_key, new_value);
+
+      // Create another clusterGroup using third cluster.
+      const newBannerMessageToAssert = /Matches 1 of 3 existing clusters: "imported-\d"/
+      cy.clickNavMenu(['Cluster Groups']);
+      cy.contains('.title', 'Cluster Groups').should('be.visible');
+      cy.createClusterGroup(newClusterGroupName, new_key, new_value, newBannerMessageToAssert);
```

Comment:
I have a question here.
Reading the steps over https://github.com/rancher/fleet-e2e/issues/181 I see on the 5th step of Fleet-29 this step: 

- Update above clusterGroup with new label and check it shows only 1 cluster 

If I understand correct this line it seems the creation of a new cluster group, but not an update. Do I understand it well and should be an update or maybe I am not having something into consideration?

> third cluster.

Reply:
Ok then. I liked the update idea, but if you are going to modify that step, could you please also update as well QASE (both title and steps) because both mention an update. Thanks